src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -70,7 +75,7 @@ export class HaAutomationEditor extends LitElement {
         .backCallback=${() => this._backTapped()}
         .tabs=${configSections.automation}
       >
-        ${this.creatingNew
+        ${this.automationId
```

Comment:
We are creating a new one if we have no automation Id ?

Reply:
I think that this should be 
```suggestion
        ${!this.automationId
```